### PR TITLE
Fix for #998

### PR DESCRIPTION
**Pull request by RoyC-Retail-Assist**
Fix with tests for #998

### CODE REVIEW

File: Tests/Linq/UserTests/Issue998Tests.cs [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+using LinqToDB.SqlQuery;
+using NUnit.Framework;
+using Tests.Model;
+
+namespace Tests.UserTests
+{
+	public class Issue998Tests : TestBase
+	{
+		[Test, IncludeDataContextSource(ProviderName.SQLiteClassic)]
+		public void AddDaysFromColumnPosative(string context)
```
Comment (sdanyliv) [inline Tests/Linq/UserTests/Issue998Tests.cs]: Typo in word positive

Comment (RoyC-Retail-Assist) [reply Tests/Linq/UserTests/Issue998Tests.cs]: Fixed

**Comment by sdanyliv** [comment]
@RoyC-Retail-Assist, it is applicable only for SQLiteClassic?

**Comment by RoyC-Retail-Assist** [comment]
I've added SQLiteMS to the contexts to test.

I would suggest that the tests should be applicable to all providers as it provides for use cases that aren't currently covered however I do not have test environments for other providers so kept the tests specific to SQLite.

**Comment by sdanyliv** [comment]
@RoyC-Retail-Assist, make it availaible for other providers also. If they have problems we will fix them.

**Comment by RoyC-Retail-Assist** [comment]
Should I move the tests into the DateTimeFunctions test class?

**Comment by sdanyliv** [comment]
Yes, please.

**Comment by RoyC-Retail-Assist** [comment]
Done :)

File: Tests/Linq/Tests.csproj [hunk 2]
```diff
@@ -58,6 +58,7 @@
 			<Link>Database/Create Scripts/%(FileName)%(Extension)</Link>
 			<CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
 		</None>
+		<Compile Remove="UserTests\Issue998Tests.cs" />
```
Comment (sdanyliv) [inline Tests/Linq/Tests.csproj]: This line should be removed

Comment (RoyC-Retail-Assist) [reply Tests/Linq/Tests.csproj]: Done

**Pull request merged**